src/packages/spinner/useSpinner.ts

```diff
@@ -1,7 +1,8 @@
 import { useSelector } from 'react-redux';
 
+import { AppState } from '../../app/store';
```

Comment:
we cant import anything from the app, this will be extracted to a separate package

Reply:
tak tam daj generic parameter... 🙄